src/api/app/views/shared/_sign_up.html.erb

```diff
@@ -10,6 +10,7 @@
       <%= text_field_tag 'login', nil, :placeholder => 'Username', :autocomplete => 'off' %><br/>
       <%= text_field_tag 'email', nil, :placeholder => 'Email address', :autocomplete => 'off' %><br/>
       <%= password_field_tag :password, nil, :id => 'pwd', :placeholder => 'Enter a password', :autocomplete => 'off' %><br />
+      <%= password_field_tag :password_confirmation, nil, :id => 'pwd_repeat', :placeholder => 'Repeat password', :autocomplete => 'off' %><br />
```

Comment:
Why? We also don't use `pwd` for `:password`. I used the same pattern as in line 12.